config/redis.yml

```
@@ -120,6 +120,9 @@ development: &defaults
   va_mobile_session_refresh_lock:
     namespace: va-mobile-session-refresh-lock
     each_ttl: 60
+  eps_appointment_access_token:
+    namespace: eps-appointment-access-token
```

would suggest renaming because EPS is not limited to appointments:

```suggestion
  eps_access_token:
    namespace: eps-access-token
```